programs/stake/src/stake_state.rs

```diff
@@ -542,8 +544,12 @@ impl<'a> StakeAccount for KeyedAccount<'a> {
         match self.state()? {
             StakeState::Initialized(meta) => {
                 meta.authorized.check(signers, StakeAuthorize::Staker)?;
+                let stake_amount = self.lamports()?.saturating_sub(meta.rent_exempt_reserve); // can't stake the rent ;)
+                if stake_amount < MINIMUM_STAKE_DELEGATION {
+                    return Err(InstructionError::StakeDelegationTooSmall);
+                }
```

Comment:
Can you please add a comment explaining why this change does not need feature-gating?

Reply:
Comment added! Can let me know what you think?